Packs/Active_Directory_Query/Integrations/Active_Directory_Query/Active_Directory_Query.py

```diff
@@ -1637,6 +1638,49 @@ def get_mapping_fields_command(search_base):
 '''
 
 
+def set_password_not_expire(default_base_dn):
+    args = demisto.args()
+    sam_account_name = args.get('username')
+    pwd_n_exp = args.get('value')
+
+    # query by sAMAccountName
+    if sam_account_name or args.get('sAMAccountName'):
+        if sam_account_name:
+            username = escape_filter_chars(sam_account_name)
+        else:
+            username = escape_filter_chars(args['sAMAccountName'])
+        query = "(&(objectClass=User)(objectCategory=person)(sAMAccountName={}))".format(username)
+
```

Comment:
Please add here an else block for handling the case that the user didn't set both username and sAMAccountName arguments. 
In the else block raise an exception with an error of missing arguments (something like "Error: missing argument!  you should provide a username or a sAMAccountName").